device/iot-device-samples/module-method-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/ModuleMethodSample.java

```diff
@@ -117,7 +117,7 @@ else if (status == IotHubConnectionStatus.CONNECTED)
      * Receives method calls from IotHub. Default protocol is to use
      * use MQTT transport.
      *
-     * @param args 
+     * @param args The connection string and the selected protocol
```

Comment:
Did you want to mention about the exceptions here instead?

Reply:
sure